### PR TITLE
[#1275] tokens, teams, projects -- stay on list page, do not go to  details

### PR DESCRIPTION
**Pull request by srenatus**
### :nut_and_bolt: Description: What code changed, and why?

For consistency, we want to send the user back to the _list_ when they've create a new entity. This change makes that happen for tokens, teams, and projects.

#### TODOs

- [x] figure out why the change for teams doesn't work
- [x] check and eventually fix cypress tests

### :chains: Related Resources

#1275 

### :+1: Definition of Done

You're sent back to the list pages.

### :athletic_shoe: How to Build and Test the Change

- start all services, rebuild the UI, exercise the UI

### :white_check_mark: Checklist

- [x] Tests added/updated?
- [ ] Docs added/updated?

### :camera: Screenshots, if applicable

### CODE REVIEW

File: .expeditor/verify_private.pipeline.yml [hunk 1]
```diff
@@ -441,7 +441,7 @@ steps:
 
   - label: ":cypress: IAMv2"
     command:
-      - IAM=v2 integration/run_test integration/tests/cypress_e2e.sh
+      - IAM=v2.0 integration/run_test integration/tests/cypress_e2e.sh
```
Comment (tylercloke) [inline .expeditor/verify_private.pipeline.yml]: 😅😅😅

File: components/automate-ui/src/app/entities/teams/team.effects.ts [hunk 2]
```diff
@@ -130,19 +128,12 @@ export class TeamEffects {
...
 
   @Effect()
-  createTeamSuccess$ = combineLatest([
-    this.actions$.pipe(ofType<CreateTeamSuccess>(TeamActionTypes.CREATE_SUCCESS)),
-    this.store$.select(iamMajorVersion).pipe(filter(identity))])
-    .pipe(
-      map(([{ payload }, version]: [CreateTeamSuccess, IAMMajorVersion]) => {
-        // Drop the user on the newly created team page.
-        const id = version === 'v1' ? payload.guid : payload.id;
-        this.router.navigate(['admin', 'teams', id]);
```
Comment (tylercloke) [inline components/automate-ui/src/app/entities/teams/team.effects.ts]: Sneaky

File: e2e/cypress/integration/ui/settings/05_project_mgmt.spec.ts [hunk 3]
```diff
@@ -6,11 +6,15 @@ interface Project {
 }
 
 describeIfIAMV2p1('project management', () => {
+  // we increase the default delay to mimic the average human's typing speed
+  // only need this for input values upon which later test assertions depend
+  // ref: https://github.com/cypress-io/cypress/issues/534
+  const typeDelay = 50;
```
Comment (bcmdarroch) [inline e2e/cypress/integration/ui/settings/05_project_mgmt.spec.ts]: tbh I'm not sure `typeDelay` is worth keeping, it hasn't really decreased flakiness in buildkite as far as I can tell. Instead we could add some `should` assertions to check that the form and save button are loaded

File: e2e/cypress/integration/ui/settings/05_project_mgmt.spec.ts [hunk 4]
```diff
@@ -207,6 +208,30 @@ describeIfIAMV2p1('project management', () => {
 
     cy.get('app-project-list chef-button').contains('Delete Project').click();
 
-    cy.get('app-project-list chef-tbody chef-td').contains(projectID).should('not.exist');
+    // empty UI should show up so entire table will be missing
```
Comment (bcmdarroch) [inline e2e/cypress/integration/ui/settings/05_project_mgmt.spec.ts]: What if other projects have been created? Let's keep it the way it was to account for that.

**Pull request merged**